packages/support/resources/js/components/action-group.js

```diff
@@ -0,0 +1,55 @@
+import {computePosition, flip, shift, autoUpdate} from '@floating-ui/dom'
+
+export default (Alpine) => {
+    Alpine.data('actionGroupTableComponent', ({
+        mountTableActionUsing
+    }) => {
+        return {
+            isOpen: false,
+
+            mountTableActionUsing,
+
+            cancelAutoUpdate: function () {},
+
+            init: function () {
+                //
+            },
+
+            update: function () {
+                const trigger = this.$refs.trigger
+                const dropdown = this.$refs.dropdown
+
+                this.cancelAutoUpdate = autoUpdate(trigger, dropdown, () => {
+                    computePosition( trigger, dropdown, {
+                        placement: 'bottom-end',
+                        middleware: [
+                            flip(),
+                            shift({ padding: 5 })
+                        ],
+                    }).then(({x, y}) => {
+                        Object.assign(dropdown.style, {
+                            left: `${x}px`,
+                            top: `${y}px`,
+                        })
+                    })
+                })
+            },
+
+            toggleDropdown: function () {
+                this.isOpen = ! this.isOpen
+
+                if( this.isOpen ){
+                    this.update()
+                } else {
+                    this.cancelAutoUpdate()
+                }
+            },
+
+            wireClickAction: function (actionName, tableRecordKey) {
```

Comment:
This is the function that replaced `wire:click="mountTableAction"`.

We need the alpine component to call $wire, since when the dropdown is teleported outside de Livewire component, it loses the Livewire reference to make the call using wire:click